DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts.cshtml

```diff
@@ -50,37 +41,64 @@
           <dt class="govuk-summary-list__key">
             Accounting officer
           </dt>
-          <dd class="govuk-summary-list__value">
-            <p class="govuk-body">Tyler Welch</p>
-            <p class="govuk-body">
-              <a class="govuk-link" href="#">Tyler.Welch@abbeylaneacademiestrust.co.uk</a>
-            </p>
-          </dd>
+          @{ DisplayContact(Model.AccountingOfficer); }
         </div>
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
             Chair of trustees
           </dt>
-          <dd class="govuk-summary-list__value">
-            <p class="govuk-body">Courtney Pacocha</p>
-            <p class="govuk-body">
-              <a class="govuk-link" href="#">Courtney.Pacocha@abbeylaneacademiestrust.co.uk</a>
-            </p>
-          </dd>
+          @{ DisplayContact(Model.ChairOfTrustees); }
         </div>
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">
             Chief financial officer
           </dt>
-          <dd class="govuk-summary-list__value">
-            <p class="govuk-body">Lowell Hoppe</p>
-            <p class="govuk-body">
-              <a class="govuk-link" href="#">Lowell.Hoppe@abbeylaneacademiestrust.co.uk</a>
-            </p>
-          </dd>
+          @{ DisplayContact(Model.ChiefFinancialOfficer); }
         </div>
 
       </dl>
     </div>
   </div>
 </section>
+
+@functions {
+
+  private void DisplayContact(Person? contactToDisplay)
+  {
+    <dd class="govuk-summary-list__value">
+
+      @if (contactToDisplay is null)
+      {
+        <p class="govuk-body">@Model.ContactInformationNotAvailableMessage</p>
+      }
+
+      else if (contactToDisplay?.FullName is not null)
+      {
+        <p class="govuk-body">@contactToDisplay.FullName</p>
+      }
+
+      else
+      {
+        <p class="govuk-body">@Model.ContactNameNotAvailableMessage</p>
+      }
+
+      <p class="govuk-body">
+        @if (contactToDisplay is null)
+        {
+          <a class="govuk-body">@Model.ContactInformationNotAvailableMessage</a>
+        }
+
+        else if (contactToDisplay?.Email is not null)
+        {
+          <a class="govuk-link" href="mailto:@contactToDisplay.Email">@contactToDisplay.Email</a>
```

Comment:
other `mailto:` links in FIAT are using `rel="noopener" target="_blank"`, we should be consistent